.github/workflows/pull_request.yml

```diff
@@ -50,8 +50,72 @@ jobs:
         run: pipenv install --dev
       - name: Test translation templates
         run: make test-translation-templates
+  translate-schemas:
+    needs: test-translation-templates
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Download built schemas
+        uses: actions/download-artifact@v1
+        with:
+          name: schemas
+      - name: Install pipenv
+        run: pip install pipenv==2018.11.26
+      - run: |
+          echo "::set-env name=PYTHON_VERSION::$(python -V 2>&1)"
+      - name: Cache virtualenv
+        id: cache-virtualenv
+        uses: actions/cache@v1
+        with:
+          path: ~/.local/share/virtualenvs/
+          key: ${{ runner.os }}-${{ env.PYTHON_VERSION }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
+      - name: Install virtualenv
+        if: steps.cache-virtualenv.outputs.cache-hit != 'true'
+        run: pipenv install --dev
+      - name: Translate schemas
+        run: make translate-schemas
+      - name: Archive schema artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: schemas
+          path: schemas
```

Comment:
There shouldn't be any need to add the translation step to the pull request workflow. This only needs to be done when the schemas are released. All that's needed for the suggestion url to be resolved is the language code?

Reply:
Are you sure? `resolve-suggestions-urls` method needs all folders to be present, otherwise there are no schemas in cy/ga/eo folders.

Reply:
resolve-suggestion-urls should only be resolving urls for schemas that are built. We only build the english one on pull-request workflows and then the others on merges to master.

Reply:
So I removed that step now and it is optional in `resolve-suggestions-urls` script. Thanks!